### PR TITLE
feat: Realize_Bipartite_Degree_Sequence()

### PR DESCRIPTION
**Pull request by szhorvat**

### CODE REVIEW

File: src/_igraph/graphobject.c [hunk 1]
```diff
@@ -2975,6 +2975,59 @@ PyObject *igraphmodule_Graph_Realize_Degree_Sequence(PyTypeObject *type,
...
+  if (igraphmodule_PyObject_to_realize_degseq_t(method_o, &method))
+    return NULL;
+
+  /* First degree vector */
+  if (igraphmodule_PyObject_to_vector_int_t(degrees1_o, &degrees1))
+    return NULL;
+
+  /* Second degree vector */
+  if (igraphmodule_PyObject_to_vector_int_t(degrees2_o, &degrees2))
+    return NULL;
```
Comment (ntamas) [inline src/_igraph/graphobject.c:3013]: At this point you already have an initialized `igraph_vector_int_t` in `degrees1` so you must destroy it before returning `NULL`. (There's no FINALLY stack in the Python interface, you need to do stuff manually).

**Review by ntamas — CHANGES_REQUESTED** [review]

**Comment by ntamas** [comment]
Still here, sitting in my hotel in Vienna, I was just about to go to bed :) Tomorrow I can take another look from Stockholm airport while I wait for my connection to Kiruna.

File: src/_igraph/graphobject.c (hunk 1 above)
Comment (szhorvat) [reply src/_igraph/graphobject.c:3014]: Thanks, fixed!

**Comment by ntamas** [comment]
Thanks a lot!

**Comment by szhorvat** [comment]
Hi @larah19, this will be released for Python soon.

Given the difficulties with the uni email, could you send me an email at szhorvat at gmail please? I have no other way to contact you than GitHub atm.

**Pull request merged**